core/src/main/java/io/grpc/internal/AutoConfiguredLoadBalancerFactory.java

```diff
@@ -97,12 +101,10 @@ public final class AutoConfiguredLoadBalancer {
     }
 
     /**
-     * Returns non-OK status if resolvedAddresses is empty and delegate lb requires address ({@link
-     * LoadBalancer#canHandleEmptyAddressListFromNameResolution()} returns {@code false}). {@code
-     * AutoConfiguredLoadBalancer} doesn't expose {@code
-     * canHandleEmptyAddressListFromNameResolution} because it depends on the delegated LB.
+     * Returns non-OK status if the delegate rejects the resolvedAddresses (e.g. if it does not
+     * support an empty list).
      */
-    Status tryHandleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
+    Status tryAcceptResolvedAddresses(ResolvedAddresses resolvedAddresses) {
```

Comment:
I don't think we need the Status any more. I think this ends up returning a boolean (and we could end up using acceptResolvedAddresses() here and make AutoConfiguredLoadBalancer an actual LB; but that can be later). All we need to do is call `scheduleExponentialBackOffInSyncContext()` in ManagedChannelImpl if this returns false.